lib/asker.js

```diff
@@ -315,7 +288,7 @@ Request.prototype.formatTimestamp = function() {
     var timers = this.getTimers();
 
     // @todo let user to use custom format
-    return ['in ', timers.network || '0', '~', timers.total, ' ms'].join('');
+    return 'in ' + (timers.network || '0') + '~' + timers.total + ' ms';
```

Comment:
It can break some metrics, I think.
I vote for keeping previous format, but in addition you can do user-customisable format.